src/lib/rules/validate-set-cookie-header/validate-set-cookie-header.ts

```diff
@@ -0,0 +1,226 @@
+/**
```

Comment:
@sarvaje just added type annotation to all the variables I can find. One question - I can't find much information about how to add type annotation to variables from array destructuring assignment like:
```
const [nameValuePair, ...directivePairs] = setCookieValue.split(';');
```
And if I change that to
```
let nameValuePair: string;
let directivePairs: Array<string>;

[nameValuePair, ...directivePairs] = setCookieValue.split(';');
```
It wouldn't pass the lint. Do you know how to add the type in this case?